src/components/Footer/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import styles from "./Footer.module.css";
+
+function Footer(): JSX.Element {
+  return (
+    <footer className={styles.footer}>
+      <small>owned by SyntaxError company limited © 2023</small>
+    </footer>
+  );
```

Comment:
You do something like 
```
const year = new Date().getFullYear();
```
and then replace it instead of 2023, this way you don't have to manually change it every year.